stemcell_builder/stages/bosh_sudoers/apply.sh

```
@@ -9,9 +9,9 @@ source $base_dir/lib/prelude_bosh.bash
 # setup sudoers to use includedir, and make sure we don't break anything
 cp -p $chroot/etc/sudoers $chroot/etc/sudoers.save
 echo '#includedir /etc/sudoers.d' >> $chroot/etc/sudoers
-run_in_bosh_chroot $chroot "visudo -c"
-if [ $? -ne 0 ]; then
-  echo "ERROR: bad sudoers file"
+if ! run_in_bosh_chroot $chroot "visudo -c"; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)